app/models/user.rb

```diff
@@ -52,7 +53,7 @@ def send_activation_email
   end
 
   def current_user? user
-    user == current_user
+    self == user
```

Comment:
self trong trường hợp này là gì ?

Reply:
self ở đây dùng để định nghĩa user, để cho hệ thống hiểu được đây là user cũng giống như this.user = user trong c#

Reply:
@mabugay1234 anh không hiểu chú nói gì luôn :(

Reply:
em trả lời lại ạ:
self ở đây được sử dụng để so sánh user trong hệ thống với user đang hoạt động.

Reply:
self ở đây là object (là current_user đang đăng nhập)